tensorflow_addons/rnn/cell.py

```diff
@@ -366,3 +366,425 @@ def _create_norm_layer(self, name):
+            kernel_initializer='glorot_uniform',
+            recurrent_initializer='orthogonal',
+            bias_initializer='zeros',
+            gamma_initializer='ones',  # NEW(!)
+            kernel_regularizer=None,
+            recurrent_regularizer=None,
+            bias_regularizer=None,
+            gamma_regularizer=None,  # NEW(!)
+            activity_regularizer=None,
+            kernel_constraint=None,
+            recurrent_constraint=None,
+            bias_constraint=None,
+            gamma_constraint=None,  # NEW(!)
+            dropout=0.,
+            recurrent_dropout=0.,
+            return_sequences=False,
+            return_state=False,
+            go_backwards=False,
+            stateful=False,
+            unroll=False,
+            **kwargs):
+        # 'implementation' warning was never relevant for LayerNormSimpleRNN
+        cell = LayerNormSimpleRNNCell(
+            units,
+            activation=activation,
+            use_bias=use_bias,
+            layernorm_epsilon=layernorm_epsilon,  # NEW(!)
+            kernel_initializer=kernel_initializer,
+            recurrent_initializer=recurrent_initializer,
+            bias_initializer=bias_initializer,
+            gamma_initializer=gamma_initializer,  # NEW(!)
+            kernel_regularizer=kernel_regularizer,
+            recurrent_regularizer=recurrent_regularizer,
+            bias_regularizer=bias_regularizer,
+            gamma_regularizer=gamma_regularizer,  # NEW(!)
+            activity_regularizer=activity_regularizer,
+            kernel_constraint=kernel_constraint,
+            recurrent_constraint=recurrent_constraint,
+            bias_constraint=bias_constraint,
+            gamma_constraint=gamma_constraint,  # NEW(!)
+            dropout=dropout,
+            recurrent_dropout=recurrent_dropout,
+            dtype=kwargs.get('dtype'),
+            trainable=kwargs.get('trainable', True))
+        super(keras.layers.SimpleRNN, self).__init__(  # call RNN's init
+            cell,
+            return_sequences=return_sequences,
+            return_state=return_state,
+            go_backwards=go_backwards,
+            stateful=stateful,
+            unroll=unroll,
+            **kwargs)
+        self.activity_regularizer = keras.regularizers.get(
+            activity_regularizer)
+        # self.input_spec = [InputSpec(ndim=3)]
+
+    # use SimpleRNN's call() method
+
+    @property
+    def layernorm_epsilon(self):
+        return self.cell.layernorm_epsilon
+
+    @property
+    def gamma_initializer(self):
+        return self.cell.gamma_initializer
+
+    @property
+    def gamma_regularizer(self):
+        return self.cell.gamma_regularizer
+
+    @property
+    def gamma_constraint(self):
+        return self.cell.gamma_constraint
+
+    def get_config(self):
+        base_config = super(keras.layers.SimpleRNN,
+                            self).get_config()  # get RNN's config
+        del base_config['cell']
```

Comment:
Why do we need to recreate the cell config here? I thought the base RNN config should just be fine for the cell part.

Reply:
done. LayerNormSimpleRNN was removed